pkg/karpenter/providers/helm.go

```diff
@@ -23,5 +23,10 @@ type HelmInstaller interface {
 	AddRepo(repoURL string, release string) error
 	// InstallChart takes a releaseName's name and a chart name and installs it. If namespace is not empty
 	// it will install into that namespace and create the namespace. Version is required.
-	InstallChart(ctx context.Context, releaseName, chartName, namespace, version string, values map[string]interface{}) error
+	InstallChart(
+		ctx context.Context,
+		releaseName, chartName, namespace, version string,
+		createNamespace bool,
+		values map[string]interface{},
+	) error
```

Comment:
mhmm perhaps worth chucking into a struct? Or can some of it been on the `helmInstaller` struct? (e.g. ctx)?

Reply:
Yeah thought about it too. 🤔 You just confirmed that it's way too much. :D

Reply:
The only problem is that I can't import anything from here, otherwise it's a circular dependency. So I'm creating a models package or something which can define the arguments. Which sounds a bit silly... But I don't really have a choice. Unless I define the interface in the same package as well. Which doesn't work again, because of the fake importing it to check validity which is something again, I can't avoid.

Reply:
where would the circular dependency be? Sounds like a smell if just adding a struct argument causes this :thinking:

Reply:
Also, I would have to rename the Installer again to something different... :D

Reply:
The fake, generates a type check. And since it's imported by the test, it can't import the same package.

I guess I can put the test into `helm_test`.

Reply:
Never mind. I messed something else up.